test/jdk/java/net/HttpURLConnection/HttpURLConnectionExpectContinueTest.java

```diff
@@ -73,7 +73,7 @@ public void startServerSocket() throws Exception {
 
         control.serverSocket = new ServerSocket();
         control.serverSocket.setReuseAddress(true);
-        control.serverSocket.bind(new InetSocketAddress("127.0.0.1", 54321));
+        control.serverSocket.bind(new InetSocketAddress("127.0.0.1", 0));
```

Comment:
While you're at it can you change the address to use `InetAddress.getLoopbackAddress()`here, and the URIBuilder in createConnection() below?

Reply:
based on Daniel's last comment and requested change then I think it would be useful to add two additional run requests to the test which set the network properties  java.net.preferIPv4Stack=true and java.net.preferIPv6Addresses=true, for completeness.
You never know what might "pop out of the woodwork"